decidim-comments/app/frontend/comments/down_vote_button.component.tsx

```diff
@@ -55,62 +55,60 @@ const downVoteMutation = require("../mutations/down_vote.mutation.graphql");
 const getCommentsQuery = require("../queries/comments.query.graphql");
 
 const DownVoteButtonWithMutation = graphql<DownVoteMutation, DownVoteButtonProps>(downVoteMutation, {
-  props: ({ ownProps, mutate }) => ({
-    downVote() {
-      if (mutate) {
-        mutate({
-          variables: {
-            id: ownProps.comment.id,
+  props: ({ ownProps, mutate }: { ownProps: DownVoteButtonProps, mutate: MutationFunc<DownVoteMutation> }) => ({
```

Comment:
This error was very weird (to me, anyways). Since it was strangely only happening to "down voting", I fixed it by changing this to look exactly like "up voting" does.